app/services/nebraska/daily/revenue_calculator.rb

```diff
@@ -39,21 +39,29 @@ def ne_special_needs_revenue
       end
 
       def ne_base_revenue
-        (hours * hourly_rate * business.ne_qris_bump) +
-          (days * daily_rate * business.ne_qris_bump)
+        (hours * hourly_rate * business.ne_qris_bump(date: @date)) +
```

Comment:
Don't need the `@` here because it's defined in the attr_reader above, so you can call it without including the instance variable indicator.